Header header

std_msgs/Float32 attenuation
std_msgs/Float32 distance
std_msgs/Int16 walls
